Submitted By: Jim Gifford (jim at linuxfromscratch dot org)
Date: 2004-07-02
Initial Package Version: 2.6.8
Origin: Jim Gifford
Upstream Status: N/A
Description: Removes DB for LFS Build
 
diff -Naur iproute2-2.6.8.orig/misc/Makefile iproute2-2.6.8/misc/Makefile
--- iproute2-2.6.8.orig/misc/Makefile	2004-07-01 22:24:22.000000000 +0000
+++ iproute2-2.6.8/misc/Makefile	2004-07-02 18:45:31.995115122 +0000
@@ -1,6 +1,6 @@
 SSOBJ=ss.o ssfilter.o
 
-TARGETS=ss nstat ifstat rtacct arpd rtstat
+TARGETS=ss nstat ifstat rtacct rtstat
 
 include ../Config
 
@@ -24,9 +24,6 @@
 rtacct: rtacct.c
 	$(CC) $(CFLAGS) $(LDFLAGS) -o rtacct rtacct.c $(LIBNETLINK) -lm
 
-arpd: arpd.c
-	$(CC) $(CFLAGS) -I$(DBM_INCLUDE) $(LDFLAGS) -o arpd arpd.c $(LIBNETLINK) -ldb
-
 rtstat: rtstat.c
 	$(CC) $(CFLAGS) $(LDFLAGS) -o rtstat rtstat.c
 
